distutils/cmd.py

```diff
@@ -330,7 +332,8 @@ def get_finalized_command(self, command: str, create: bool = True) -> Command:
         'command', call its 'ensure_finalized()' method, and return the
         finalized command object.
         """
-        cmd_obj = self.distribution.get_command_obj(command, create)
+        # TODO: Raise a more descriptive error when cmd_obj is None ?
+        cmd_obj = cast(Command, self.distribution.get_command_obj(command, create))
```

Comment:
Decision to be taken here.

Reply:
I just did a quick scan through the distutils and setuptools codebase and there's not a single call to `get_finalized_command(..., create=False)`. Maybe the create flag is just cruft that can be eliminated.